smart_card_connector_app/src/background.js

```diff
@@ -166,44 +166,18 @@ if (GSC.ExecutableModule.TOOLCHAIN ===
   GSC.BackgroundPageUnloadPreventing.enable();
 }
 
-/** @type {GSC.ConnectorApp.ChromeApiProvider?} */
-let chromeApiProvider = null;
-
 if (chrome.smartCardProviderPrivate !== undefined) {
-  updateChromeApiProviderAvailability();
-  chrome.storage.onChanged.addListener(updateChromeApiProviderAvailability);
+  // This object's lifetime is bound to the message channel's one, hence we
+  // don't need to store it in a variable.
+  new GSC.ConnectorApp.ChromeApiProvider(
```

Comment:
I feel like I already asked this 10 times in the past, but keep forgetting the answer... What's the story with the backwards-incompatible changes in the API that were made by Chrome a few versions ago? Will the rest of the Connector's functionality work fine there?

Reply:
So the interesting use case is those semi-old versions on the dev channel where smartCardProviderPrivate is already available but not fully implemented.
There the app will subscribe to all the events that are implemented (for the rest it'll just print some warning messages).
And then if someone will enable the SmartCard feature flag and try to send some requests from IWA, I guess there could be some errors for GetStatusChange requests. 
But I don't think someone will ever do this :)

Reply:
Ack, sounds fine. Thanks for capturing this in the comment thread here.